src/main/java/org/fiximulator/ui/FIXimulatorFrame.java

```diff
@@ -1031,6 +1032,18 @@ public void actionPerformed(ActionEvent evt) {
             }
         });
 
+        sendLiquidityFlag.setText("Send LiquidityFlag (9882)");
```

Comment:
Consider i18n here.

Reply:
Internationalization is an important step in making the simulator available to the greater population. Might not be worth the implementation cost given the small footprint.